gr-filter/python/filter/design/filter_design.py

```diff
@@ -19,29 +19,21 @@
 from optparse import OptionParser
 
 from gnuradio import filter
+from gnuradio.fft import fft_pylib as fft_detail
 
 try:
     import numpy as np
 except ImportError:
     raise SystemExit('Please install NumPy to run this script (https://www.np.org/)')
 
-try:
-    from numpy.fft import fftpack as fft_detail
-except ImportError:
-
-    print('Could not import fftpack, trying pocketfft')
-    # Numpy changed fft implementation in version 1.17
-    # from fftpack to pocketfft
-    try:
-        from numpy.fft import pocketfft as fft_detail
-    except ImportError:
-        raise SystemExit('Could not import fft implementation of numpy')
-    
 try:
     from scipy import poly1d, signal
 except ImportError:
     raise SystemExit('Please install SciPy to run this script (https://www.scipy.org)')
 
+if not fft_detail:
+    raise SystemExit('Unable to load fft library (numpy, scipy)')
```

Comment:
Wouldn't it make sense to reword to something like the following?

> Unable to load fft library from either numpy or scipy.

Reply:
If we move the failure to the `__init__` file (see above) then we never have to do this in tools.